moveit_setup_assistant/src/widgets/default_collisions_widget.cpp

```diff
@@ -153,6 +153,7 @@ DefaultCollisionsWidget::DefaultCollisionsWidget( QWidget *parent,
   collision_table_ = new QTableWidget( this );
   collision_table_->setColumnCount(4);
   collision_table_->setSortingEnabled(true);
+  collision_table_->setSelectionMode( QAbstractItemView::SingleSelection );
```

Comment:
Actually, it would be cool to enable multiple selections and then enable/disable collisions on the whole selection at once. But, I see your point. Currently it's only meaningful to select exactly one line.